### ISSUE TITLE
Colours in Dark mode

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#180]
The colours in dark mode needs to match the prototype on :

1. Level -2 : The font colour for subject, date and share this publication button
2. For you : The background colour is not switching to the darker colour and on the saved segment the heart icon is not visible. The subject and the date colour applies to this as well. Also, the font colour is not changing for Following and Saved segement (selected and unselected).
3. More options sub-menu on For you : The colour is not matching for manage subjects
4. Voiceover hover colour: When in dark mode, please change the colour of hover to white. And, when in light mode, the hover colour needs to be changed to white.

**Comment by Jiandong-Guo** [comment]
we don't control the VoiceOver, its behavior is based on OS, we don't change OS setting.  
dark color was changed, Please test it in next release

**Comment by Rajat-Sen** [comment]
Hi Jiandong, the sub-menu items are not accessible. Can you make sure when we are using voiceover, the menu items should get Focus as well.

**Comment by Pharste** [comment]
Hi, 

Adding to this ticket since these issues were found in dark mode:

The Publications (Level 2) screen loads using the light mode theme: 
![Dark mode Pubs](https://user-images.githubusercontent.com/89025880/149441365-38406833-c629-4e3c-8cd5-b83d1b073898.jpg)

The heart icon is appearing in light theme? (I believe this is part of the issue above, but documenting with screenshot in case it's different):
![Heart icon Dark Mode](https://user-images.githubusercontent.com/89025880/149441804-3b1d0b75-dc98-4877-8ba1-ccf762ea7f93.jpg)

Following screen appearing in light mode:

![Notifications dark mode](https://user-images.githubusercontent.com/89025880/149442299-06682a33-db20-4a6e-9993-9903f8ec0aca.jpg)
![Dark mode following](https://user-images.githubusercontent.com/89025880/149442520-0fa984e5-ed4b-4a60-a12a-be3cf45042ea.jpg)

Search page, there's a grey frame showing: 

![Dark mode search](https://user-images.githubusercontent.com/89025880/149442813-fe4b7889-c479-4f6e-afd3-a7c5000a9129.jpg)

**Comment by Pharste** [comment]
The heart icon is visible now, but doesn't match the prototype: 
![Saved dark mode](https://user-images.githubusercontent.com/89025880/149589630-4d49157f-0f13-4018-baaf-ca8bf0a9591b.jpg)

The select subjects notifications screen still has a grey bar at the bottom of the screen: 
![Select subjects dark mode](https://user-images.githubusercontent.com/89025880/149590021-6d77bcb3-9040-4e5d-9b14-5a09b14015a2.png)

The dark mode search has a grey line, and darker bar at top not matching prototype. The search bar is missing the icon, text in the bar, and the bar is darker than in the prototype: 
![Search Dark mode](https://user-images.githubusercontent.com/89025880/149590294-bab07a13-1dce-4cdb-9863-03208fbc3c67.jpg)

All other screens seem to be showing properly (except for the loading which cannot be fixed).

**Comment by Pharste** [comment]
I also noticed when you click on see more from the following screen, it isn't matching the prototype: (Date is darker and the title of the publication seems smaller)
![Subject pubs list](https://user-images.githubusercontent.com/89025880/149705267-0f216d5f-8115-4e00-950f-ea1aa951f4bb.jpg)

**Comment by ForumG-SC** [comment]
For You:

Following Screen: Grey bar at the bottom removed in dark mode
Saved Screen: Heart icon color changed to transparent in dark mode
See more Screen: More Icon color changed to white in dark mode, date color changed to white in dark mode.
Search Screen: Search publications text color changed to white in dark mode.

Please validate these changes in new deployment.

@Pharste

**Comment by Rajat-Sen** [comment]
Hi Forum, on the Publication screen (Level-2) when tapped on the heart icon the colour is not matching the prototype. Should I create a separate ticket for that?

**Comment by ForumG-SC** [comment]
> Hi Forum, on the Publication screen (Level-2) when tapped on the heart icon the colour is not matching the prototype. Should I create a separate ticket for that?

It has been fixed, I forgot to mention in my comment above.

**Comment by Rajat-Sen** [comment]
Thanks!

**Comment by Pharste** [comment]
After uninstalling and reinstalling the app, I'm still seeing the heart icon with the white background on the For you - saved page: 
![Heart dark mode](https://user-images.githubusercontent.com/89025880/149826206-578624cc-4253-4a8f-a3b7-e5596084f685.jpg)

The image is overlapping the Go to publications button on iPad (but not on my phone):
![Overlapping image and button iPad](https://user-images.githubusercontent.com/89025880/149826235-d32baec6-f330-40e3-abc3-7d70da8649b5.jpg)

After completing a search, the sort and filter buttons are showing in the light theme (when in dark mode): 
- this also shows how the text is not lined up on iPad
![Light buttons search screen](https://user-images.githubusercontent.com/89025880/149826976-a6e7808b-b2cb-48b1-bd60-8b618ac2d844.jpg)

**Comment by ForumG-SC** [comment]
> After uninstalling and reinstalling the app, I'm still seeing the heart icon with the white background on the For you - saved page: ![Heart dark mode](https://user-images.githubusercontent.com/89025880/149826206-578624cc-4253-4a8f-a3b7-e5596084f685.jpg)
> 
> The image is overlapping the Go to publications button on iPad (but not on my phone): ![Overlapping image and button iPad](https://user-images.githubusercontent.com/89025880/149826235-d32baec6-f330-40e3-abc3-7d70da8649b5.jpg)
> 
> After completing a search, the sort and filter buttons are showing in the light theme (when in dark mode):
> 
> * this also shows how the text is not lined up on iPad
>   ![Light buttons search screen](https://user-images.githubusercontent.com/89025880/149826976-a6e7808b-b2cb-48b1-bd60-8b618ac2d844.jpg)

Please wait for new deployment to test this again.

**Comment by Pharste** [comment]
Hi, 

I've validated that all of the issues have been resolved. However, now on the French saved screen, the publications button is cut off at the end: 
![saved french](https://user-images.githubusercontent.com/89025880/150426510-e3df17a2-1e22-4bb7-b0fe-76709e1d2f9e.jpg)

**Comment by ForumG-SC** [comment]
Fixed

**Comment by Pharste** [comment]
Valildated, thank you
